platform/ui/src/components/HotkeysPreferences/hotkeysValidators.js

```diff
@@ -62,7 +63,7 @@ const disallowedValidator = ({ pressedKeys = [] }) => {
 
   if (hasDisallowedCombinations) {
     return {
-      error: `"${formatPressedKeys(pressedKeys)}" shortcut combination is not allowed`,
+      error: i18n.t('HotkeysValidators:Shortcut combination is not allowed', {pressedKeys: formatPressedKeys(pressedKeys)}),
```

Comment:
Same behavior as above

Reply:
👍

Reply:
I'll revert this too if interpolation is not needed

Reply:
Please keep this change, that's why I gave the thumbs up!